src/Microsoft.ComponentDetection.Common/PathUtilityService.cs

```diff
@@ -64,6 +64,7 @@ public bool IsRunningOnWindowsContainer
     /// <param name="output"> The pointer output. </param>
     /// <returns> A pointer <see cref= "IntPtr"/> to the absolute path of a file. </returns>
     [DllImport("libc", EntryPoint = "realpath")]
+    [DefaultDllImportSearchPaths(DllImportSearchPath.SafeDirectories)]
```

Comment:
How did you decide the search path here and below?

Reply:
There's no documentation of the `DllImportSearchPath` for Linux ([this search](https://github.com/search?q=repo%3Adotnet%2Fruntime%20SafeDirectories&type=code) is unhelpful as well). libc is in `/usr/lib` and I'd assume that is a "safe directory." I've searched other code that imports libc and nothing adds the `DefaultDllImportSearchPaths`. 

For `kernel32.dll` P/Invoke's, `kernel32.dll` should only be in `System32` (or perhaps `SysWoW64` on 32-bit applications). This is what [other MSFT Repos do](https://github.com/microsoft/CopyOnWrite/blob/64628b58124b357a7d9c7fed1cfda4bfa38ff654/lib/Windows/NativeMethods.cs#L122).